pom.xml

```diff
@@ -72,9 +72,9 @@
         <jaxb-api.version>2.3.1</jaxb-api.version>
         <jaxws-rt.version>2.3.3</jaxws-rt.version>
         <jboss-logging.version>3.4.2.Final</jboss-logging.version>
-        <jdbi3.version>3.21.0</jdbi3.version>
+        <jdbi3.version>3.25.0</jdbi3.version>
         <jersey.version>2.33</jersey.version>
-        <jetty.version>9.4.43.v20210629</jetty.version>
+        <jetty.version>9.4.44.v20210927</jetty.version>
         <joda-time.version>2.10.10</joda-time.version>
```

Comment:
DW 2.0.26 uses Joda 2.10.13...I am guessing we are going to need to go through every single dependency in this pom and compare its version against what DW 2.0.26 uses, directly or via transitive dependency.

Reply:
See my previous comments